### DISCUSSION TITLE
Link tracking trace merge

### DISCUSSION THREAD

**Discussion by peoplewhoworkhard** [apache/skywalking#11306] [Q&A]
Is there any ready-made plugin for skywalking that can merge multiple traces into one?
---
My requirement is: Some links are found to be incomplete, and I want to make the links complete by merging links. For example, according to serviceId and endpointId, 10 traces are detected, and then these 10 traces are deduplicated and merged into one.

I hope to get an answer, thank you very much

**Comment by wu-sheng** [comment]
There is no such thing.
